srcpkgs/xwallpaper/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'xwallpaper'
 pkgname=xwallpaper
-version=0.5.0
+version=0.6.0
 revision=2
```

Comment:
revision needs to be set to 1 when a new version appears.

Reply:
Done, thanx!